src/site/markdown/index.md.vm

```diff
@@ -59,4 +60,6 @@ Getting Started
 mvn package
 ```
 
+See the "Goals" page for more details about the supported configuration parameters.
+
```

Comment:
Changing this back so I can merge the other change. If it requires changing, it can be done in a separate PR.

```suggestion
```